src/main/kotlin/earth/terrarium/cloche/api/metadata/FabricMetadata.kt

```diff
@@ -40,24 +22,14 @@ interface FabricMetadata {
         entrypoint(name, listOf(action))
 
     fun entrypoint(name: String, actions: List<Action<Entrypoint>>) {
+        val mergedEntrypoints = this.entrypoints.get().toMutableMap()
```

Comment:
Do not eagerly resolve lazy values. Especially in this way which erases their dependencies/context.